test/e2e/framework/helpers/k8s/k8s_utils.go

```diff
@@ -141,6 +191,61 @@ func (k *KubeCtl) GetPodNamesFromNS(namespace string) ([]string, error) {
 	return s, nil
 }
 
+func (k *KubeCtl) PortForwardPod(req PortForwardAPodRequest) error {
+	path := fmt.Sprintf("/api/v1/namespaces/%s/pods/%s/portforward",
+		req.Pod.Namespace, req.Pod.Name)
+	hostIP := strings.TrimLeft(req.RestConfig.Host, "htps:/")
+
+	transport, upgrader, err := spdy.RoundTripperFor(req.RestConfig)
+	if err != nil {
+		panic(err)
+	}
+
+	dialer := spdy.NewDialer(upgrader, &http.Client{Transport: transport}, http.MethodPost, &url.URL{Scheme: "https", Path: path, Host: hostIP})
+	fw, err := portforward.New(dialer, []string{fmt.Sprintf("%d:%d", req.LocalPort, req.PodPort)}, req.StopCh, req.ReadyCh, req.Streams.Out, req.Streams.ErrOut)
+	if err != nil {
+		return err
+	}
+	return fw.ForwardPorts()
+}
+
+func (k *KubeCtl) PortForwardYkSchedulerPod() {
+	stopCh := make(chan struct{}, 1)
+	readyCh := make(chan struct{})
+	stream := genericclioptions.IOStreams{
+		In:     os.Stdin,
+		Out:    os.Stdout,
+		ErrOut: os.Stderr,
+	}
+
+	go func() {
+		schedulerPodName, err := k.GetSchedulerPod()
+		if err != nil {
+			panic(err)
```

Comment:
Instead of panicking out, when there is an error happened, can we notify the main process and exit with an error? Then the caller can catch this exception and print some more meaningful error messages.